jh/Cargo.toml

```diff
@@ -0,0 +1,22 @@
+[package]
+name = "jh"
+version = "0.1.0"
+authors = ["RustCrypto Developers"]
+license = "MIT or Apache-2.0"
+description = "Pure Rust implementation of JH"
+documentation = "https://docs.rs/jh"
+repository = "https://github.com/RustCrypto/hashes"
+keywords = ["crypto", "jh", "hash", "digest"]
+categories = ["cryptography", "no-std"]
+edition = "2021"
+rust-version = "1.57"
+
+[dependencies]
+block-buffer = "0.10"
+block-padding = "0.3"
```

Comment:
I don't think you use `block-padding` anywhere in the code. `block-buffer` can be imported from the `digest` crate.